content/flux/v0.x/release-notes.md

```diff
@@ -10,6 +10,30 @@ aliases:
   - /influxdb/cloud/reference/release-notes/flux/
 ---
 
+## v0.139.0 [2021-11-01]
+
+### Features
+- Always continue type inference on error.
```

Comment:
may read a bit like a bug fix... could we say Add support for type inference on error? would this apply to all functions?

Reply:
Yes, this applies to all functions. I'll reword this release not a bit. The reason this is a feature is Flux will now report all type inference errors in a script. Before, it'd stop on the first error and report it. You'd then have to run the script again to get to the next error; then again for the next error. This just reports everything in one shot. Super nice.